Kickstarter-iOS/Views/Cells/PledgeCreditCardView.swift

```diff
@@ -108,6 +105,9 @@ final class PledgeCreditCardView: UIView {
   override func bindStyles() {
     super.bindStyles()
 
+    _ = self
```

Comment:
I don't even think we need these lines. Noticed that we usually don't set `self.subViews` as accessibilityElements. I tested without them and it worked just fine 🤷🏻‍♂️